gulpfile.js

```diff
@@ -137,17 +137,14 @@ gulp.task('requirejs', function(done) {
     }, done);
 });
 
-gulp.task('cloc', ['build'], function() {
+gulp.task('cloc', ['clean'], function() {
     var cmdLine;
-    var clocPath = path.join('Tools', 'cloc-1.60', 'cloc-1.60.pl');
-    var cloc_definitions = path.join('Tools', 'cloc-1.60', 'cloc_definitions');
+    var clocPath = path.join('node_modules', 'cloc', 'lib', 'cloc');
 
     //Run cloc on primary Source files only
     var source = new Promise(function(resolve, reject) {
-        var glsl = globby.sync(['Source/Shaders/*.glsl', 'Source/Shaders/**/*.glsl']).join(' ');
-
-        cmdLine = 'perl ' + clocPath + ' --quiet --progress-rate=0 --read-lang-def=' + cloc_definitions +
-                  ' Source/main.js Source/Core/ Source/DataSources/ Source/Renderer/ Source/Scene/ Source/Widgets/ Source/Workers/ ' + glsl;
+        cmdLine = 'perl ' + clocPath + ' --quiet --progress-rate=0' +
```

Comment:
Just to clarify, we run this after `clean` so this only counts the `.glsl` shaders and not the generated `.js` ones, correct? I don't think blowing away the built files will be an expected consequence of running this task, can we just exclude the built files?

Also is it still necessary to run this with perl? I think you should be able to call the executable from `node_modules` directly.

Reply:
cloc isn't an executable, it's a perl script

yes, clean gets rid of the `js` files so we're only counting the `glsl` shaders

this is the last step in the travis build, so there shouldn't be any problems running `clean`

Reply:
@ggetz see #6161, we can't easily ignore them without larger structure changes.

Reply:
OK with me then.